Formula/gtk-vnc.rb

```diff
@@ -1,8 +1,9 @@
 class GtkVnc < Formula
   desc "VNC viewer widget for GTK"
   homepage "https://wiki.gnome.org/Projects/gtk-vnc"
-  url "https://download.gnome.org/sources/gtk-vnc/1.0/gtk-vnc-1.0.0.tar.xz"
-  sha256 "a81a1f1a79ad4618027628ffac27d3391524c063d9411c7a36a5ec3380e6c080"
+  url "https://download.gnome.org/sources/gtk-vnc/1.2/gtk-vnc-1.2.0.tar.xz"
+  sha256 "7aaf80040d47134a963742fb6c94e970fcb6bf52dc975d7ae542b2ef5f34b94a"
+  license "LGPL-2.1-or-later"
```

Comment:
Based on license info I saw:
- `meson.build` has `license: 'LGPLv2.1+'`
- `COPYING.LIB` (modified 9 years ago) is for LGPL-2.1
- `Changelog` on 2017-08-08 mentions switching LGPL from v2-only to v2-or-later
- Code in `src/` shows LGPLv2.0 or later:
    ```
     * modify it under the terms of the GNU Lesser General Public
     * License as published by the Free Software Foundation; either
     * version 2.0 of the License, or (at your option) any later version.
    ```